packages/pangea-sdk/PangeaCyber.Net/AuthN/Clients/User.cs

```diff
@@ -49,54 +49,69 @@ public async Task<Response<UserCreateResult>> Create(UserCreateRequest request)
             return await DoPost<UserCreateResult>("/v2/user/create", request);
         }
 
-        /// <kind>method</kind>
         /// <summary>Delete a user by email address.</summary>
         /// <remarks>Delete User - Email</remarks>
         /// <operationid>authn_post_v2_user_delete 1</operationid>
-        /// <param name="email" type="string">An email address</param>
-        /// <returns>Response&lt;UserDeleteResult&gt;</returns>
+        /// <param name="email">An email address</param>
+        /// <returns>An empty object.</returns>
         /// <example>
         /// <code>
         /// await client.User.DeleteByEmail("joe.user@pangea.cloud");
         /// </code>
         /// </example>
-        public async Task<Response<UserDeleteResult>> DeleteByEmail(string email)
```

Comment:
I chose to use an empty class just thinking that, maybe,  in the future we would want to return something and it wouldn't be a breaking change for users. Here they should switch from `object` to `UserDeleteResult`, am I wrong?

Reply:
Added them back.